utils/design-vars.stanza

```diff
@@ -13,3 +18,59 @@ public var PREFERRED-MOUNTING = "smd" ; values in ["smd", "through-hole"]
 public var MIN-CAP-VOLTAGE = 10.0 ; Minimum voltage to allow for capacitors
 ; For consolidation
 ;public var PREFERRED-MANUFACTURERS = ["Yageo", "Panasonic Electronic Components", "Vishay Dale", "TE Connectivity Passive Product"]
+
+
+;------------------------------------------------------------------------------
+; Design variables used to configure land pattern generation/checks
+;------------------------------------------------------------------------------
+
+; Corresponds to IPC 6011
+public defenum DesignProducability:
+  LevelA
+  LevelB
+  LevelC
+
+; The construction type of the design (rigid, flex)
+public defenum DesignConstructionType:
+  Rigid
+  Flex
+
+; The application domain of the design (consumer, automative, aerospace)
+public defenum DesignApplicationType:
+  Consumer,
+  Automotive,
+;   Aerospace
+
+; The Design Producability corresponds to the IPC 6011 spec for design producability.
+; These correspond to tolerances, measurements, assembly and testing or verification 
+; standards for the designs.
+public var DESIGN-PRODUCIBILITY: DesignProducability = LevelA
+
+; The design construction type refers to the kind of board you are trying to design. 
+; The options are Rigid() or Flex().
+public var DESIGN-CONSTRUCTION-TYPE: DesignConstructionType = Rigid
+
+; The design application type refers to the domain in which your device will function. 
+; The options are False (none), Consumer(), Automotive(), and Aerospace()
+public var DESIGN-APPLICATION-TYPE: False|DesignApplicationType = false
```

Comment:
I would defer this kind of flag for now until we have a clear path for how it's used